src/hackathons-collector/logic-components/listing-item.test.js

```diff
@@ -1 +1,34 @@
-test.todo('test');
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
+import moment from 'moment';
+import 'moment/locale/en-gb';
```

Comment:
we don't need that actually here actually, it would be worth it to do that in `setupTests`